gui/src/components/maintenance/cortx-maintenance-menu.vue

```diff
@@ -47,15 +47,15 @@ export default class CortxMaintenanceMenu extends Vue {
       actionClickFunction: "goToFirmware",
       disabled: false,
       requiredAccess: "maintenance",
-      featureId: "FW_Update"
+      featureId: "fw_update"
```

Comment:
Please use from const file(unsupported-feature.ts). Otherwise we are missing the benefits of creating constant file.

Reply:
Updated